protocol/x/clob/e2e/liquidation_deleveraging_test.go

```diff
@@ -694,134 +694,6 @@ func TestPlacePerpetualLiquidation_Deleveraging(t *testing.T) {
 				},
 			},
 		},
-		`Can place and match liquidation order for subaccount with non-linear margin requirement`: {
```

Comment:
It seems like we are mostly deleting unit tests. Do we have a test where a subaccount would've been liquidated under non-linear margin requirements, but is collateralized after this change?